File: CS/PdfViewerSample/Form1.cs
using System;
using System.Windows.Forms;

namespace PdfViewerSample {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            this.pdfViewer1.LoadDocument(@"..\..\Report.pdf");
        }
    }
}

File: VB/PdfViewerSample/Form1.vb
Imports System
Imports System.Windows.Forms

Namespace PdfViewerSample
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            Me.pdfViewer1.LoadDocument("..\..\Report.pdf")
        End Sub
    End Class
End Namespace
